R/vulnerability/disasters-emergencies/flooding/2022-interim/england/mobility/private-transport.R

```diff
@@ -0,0 +1,28 @@
+library(readr)
+library(tidyr)
+library(dplyr)
+library(readxl)
+library(stringr)
+library(geographr)
+
+caci_demographics <- 
+  read_excel(
+    "C:/Users/040026704/British Red Cross Society/Strategic Insight and Foresight - Documents/Data/CACI/BRCS_LSOA_Up to Date Demographics_July_2021.xlsx",
+    skip = 10
+  )
+
+lack_of_private_transport <- 
+  caci_demographics %>% 
+  filter(str_detect(LSOA, "^E")) %>% 
+  select(lsoa_code = LSOA, no_cars = `No Cars Or Vans`, population = `Total Population...2`) %>% 
```

Comment:
Think the denominator is number of households rather than population. 
```suggestion
  select(lsoa_code = LSOA, no_cars = `No Cars Or Vans`, population = `Total Households...55`) %>%
```